### PR TITLE
[serverles-1.31] Change default repo coordinates to midstream

### PR DESCRIPTION
**Pull request by dsimansk**
Just in case, to be on a safe side. 

/cc @matejvasek @jrangelramos

### CODE REVIEW

File: pkg/pipelines/tekton/templates.go [hunk 1]
```diff
@@ -88,8 +88,8 @@ const (
 )
 
 var (
-	FuncRepoRef       = "knative/func"
-	FuncRepoBranchRef = "main"
+	FuncRepoRef       = "openshift-knative/kn-plugin-func"
+	FuncRepoBranchRef = "serverless-1.30"
```
Comment (jrangelramos) [inline pkg/pipelines/tekton/templates.go:92]: shoudn't it be "serverless-1.31" ?

Comment (dsimansk) [reply pkg/pipelines/tekton/templates.go:92]: 🤦 thanks!

Comment (dsimansk) [reply pkg/pipelines/tekton/templates.go:92]: I'll create another PR, sorry about that. Too close numbers probably.

**Pull request merged**